container/libcontainer/helpers.go

```diff
@@ -203,12 +208,23 @@ func schedulerStatsFromProcs(rootFs string, pids []int) (info.CpuSchedstat, erro
 			switch i {
 			case 0:
 				schedstats.RunTime += metric
```

Comment:
You dont need to add to schedstats here.  Just add to cacheEntry, and aggregate at the end.